tests/sles4sap/hana_install.pm

```diff
@@ -242,7 +242,10 @@ sub run {
     assert_script_run "df -h";
 
     # hdblcm is used for installation, verify if it exists
-    my $hdblcm = '/sapinst/' . get_var('HANA_HDBLCM', "DATA_UNITS/HDB_SERVER_LINUX_" . uc(get_required_var('ARCH')) . "/hdblcm");
+    my $hdblcm = '/' . join('/',
```

Comment:
Isn't this preppending an additional `/` to the command? `$target` is already `/sapinst`.

Reply:
This file should not be part of this PR at all